src/components/WebLicenseCode.vue

```diff
@@ -0,0 +1,225 @@
+            currentTab: 0,
+            copyText: this.$t('license-use.copy-label'),
+            currentSelection: 'richtext'
+        }
+    },
+    methods: {
+        onCopySuccess(e) {
+            this.success = true
+            if (process.env.NODE_ENV === 'production') {
+                const fieldsFilled = {}
+                for (const detail in this.attributionDetails) {
+                    fieldsFilled[detail] = this.attributionDetails[detail] !== ''
+                }
+                const copiedLicense = {
+                    license: this.shortName,
+                    // codeType can be either rich-text or html
+                    codeType: this.currentSelection,
+                    fieldsFilled: fieldsFilled
+                }
+                this.$ga.event({
+                    eventCategory: 'Attribution',
+                    eventAction: 'copied',
+                    eventLabel: JSON.stringify(copiedLicense)
+                })
+            }
+            this.$emit('copied', { content: e.text })
+            setTimeout(() => {
+                this.success = false
+            }, 2000)
+            e.clearSelection()
+        },
+        onCopyError(e) {
+            this.$emit('copyFailed')
+            e.clearSelection()
+        },
+        clipboardTarget() {
+            return `#attribution-${this.currentSelection}`
+        }
+    },
+    computed: {
+        ...mapGetters(['shortName', 'fullName', 'iconsList', 'licenseUrl']),
+        attributionDetails() {
+            return this.$store.state.attributionDetails
+        },
+        htmlLicenseParagraph() {
+            const data = generateHTML(this.attributionDetails, this.shortName)
+            const licenseCodeSpan = this.$i18n.t('license-use.richtext.full-text', {
+                workTitle: data.workTitle ? data.workTitle : this.$i18n.t('license-use.richtext.workTitle'),
+                creator: data.creator,
+                licenseName: this.shortName,
+                by: data.creator ? this.$i18n.t('license-use.richtext.by') : ''
+            })
+            return `${data.htmlString}${licenseCodeSpan}${data.licenseIconsLink}</p>`
+        },
+        activeTab: {
+            get() { return this.currentTab },
+            set(val) {
+                if (val !== 2) {
+                    this.currentTab = val
+                } else {
+                    this.currentSelection = this.currentTab === 0 ? 'richtext' : 'html'
+                    const tab = this.currentTab
+                    this.currentTab = 2
+                    this.copyText = this.$t('license-use.copied-label')
+                    setTimeout(() => {
+                        this.currentTab = tab
+                    }, 1)
+                    setTimeout(() => {
+                        this.copyText = 'Copy'
+                    }, 2000)
+                }
+            }
+        }
+    },
+    mounted() {
+        this.clipboard = new Clipboard('.copyBtn')
```

Comment:
Creating a separate instance of `Clipboard` for both Web and PrintLicenseCode components may cause issues. This is probably not a big deal, but I think it would be good to try and find an elegant way to have one instance of `Clipboard` that works for both Web and PrintLicenseCode

Reply:
I think we should address this, but I'm resolving it with the expectation that it will be addressed somehow in the future.